inflate.c

```diff
@@ -1504,7 +1504,7 @@ z_streamp strm;
 {
     struct inflate_state FAR *state;
 
-    if (strm == Z_NULL || strm->state == Z_NULL) return -1L << 16;
+    if (strm == Z_NULL || strm->state == Z_NULL) return (unsigned)-1L << 16;
```

Comment:
This won't return the same value as without the cast on LP64 platforms, where ((unsigned)-1L) != ((long)-1L)